centos-ci/bootstrap.sh

```diff
@@ -11,21 +23,29 @@ yum install -y python2-pip gcc redhat-rpm-config openssl-devel python-devel
 
 # Interim EPEL-based approach to enable testing of --ask-pass option
 yum install -y sshpass
+time_since_begining "step1: install yum deps"
```

Comment:
Should we adjust the helper function to also reset the start time variable, so each message displays the time since the previous message, rather than the time since the beginning of the run?